api/source/mk_events_objects.cpp

```diff
@@ -182,7 +182,7 @@ API_EXPORT const char* API_CALL mk_media_source_get_vhost(const mk_media_source
 }
 API_EXPORT const char* API_CALL mk_media_source_get_app(const mk_media_source ctx){
     assert(ctx);
-    MediaSource *src = (MediaSource *)ctx;
+    MediaSource  *src = (MediaSource *)ctx;
```

Comment:
extra useless white space

Reply:
Thank you for the reminder. It has been removed